gui/mainFrame.py

```diff
@@ -142,7 +142,7 @@ class MainFrame(wx.Frame):
 
     @classmethod
     def getInstance(cls):
-        return cls.__instance if cls.__instance is not None else MainFrame()
+        return cls.__instance if cls.__instance is not None else MainFrame("")
```

Comment:
I removed this and added a default to the main frame definition. Makes more sense.